oak-core/src/main/java/org/apache/jackrabbit/oak/query/SQL2Parser.java

```diff
@@ -676,6 +676,12 @@ private DynamicOperandImpl parseExpressionFunction(String functionName) throws P
             } else {
                 op = factory.nodeLocalName(readName());
             }
+        } else if ("PATH".equalsIgnoreCase(functionName)) {
```

Comment:
I think you should add `PATH` and `FIRST` in the `getSyntaxError ` at line 707

Reply:
done